cmd/minikube/cmd/start_flags.go

```diff
@@ -780,6 +789,14 @@ func updateExistingConfigFromFlags(cmd *cobra.Command, existing *config.ClusterC
 		out.WarningT("You cannot change the static IP of an existing minikube cluster. Please first delete the cluster.")
 	}
 
+	if cmd.Flags().Changed(qemuCPUType) && viper.GetString(qemuCPUType) != existing.QEMUCPUType {
+		out.WarningT("You cannot change the QEMU CPU type of an existing minikube cluster. Please first delete the cluster.")
+	}
+
+	if cmd.Flags().Changed(qemuMachineType) && viper.GetString(qemuMachineType) != existing.QEMUMachineType {
+		out.WarningT("You cannot change the QEMU machine type of an existing minikube cluster. Please first delete the cluster.")
+	}
```

Comment:
Maybe QEMU program should also not be allowed to be changed on an existing cluster?